NCAR/main.yaml

```diff
@@ -35,9 +35,66 @@ sources:
       simulation_id: unknown
       time_start: 2020-01-20T00:00:00
       time_end: 2020-03-05T00:00:00
-      summary: Atmosphere-land simulation at 3.75km grid spacing. Run from 2020-01-20 to 2020-03-05.
+      summary: |
+         Atmosphere-land simulation at 3.75km grid spacing. Run from 2020-01-20 to 2020-03-05.
+
+         **Resolutions**
+           * All data is available at HEALPix levels 1-10 for 2D fields
+
+         **Processing**
+           * Fields were remapped to HEALPix level 10 using nearest-neighbor interpolation. 
+           * Lower levels were generated from this using coarse-graining.
+         
       creator_name: Bill Skamarock
       institution: NSF National Center for Atmospheric Research
+  mpas_dyamond1:
+    args:
+      chunks: null
+      consolidated: true
+      urlpath: /glade/derecho/scratch/digital-earths-hackathon/mpas_DYAMOND1/{{time}}/DYAMOND1_diag_{{time}}_to_hp{{zoom}}.zarr
+    driver: zarr
+    parameters:
+      time:
+        allowed:
+        - 15min
+        default: 15min
+        description: temporal resolution of the dataset
+        type: str
+      zoom:
+        allowed:
+        - 10
+        - 9
+        - 8
+        - 7
+        - 6
+        - 5
+        - 4
+        - 3
+        - 2
+        - 1
+        default: 7
```

Comment:
For ICON we usually set the default zoom level to 0. The reason is that this causes the least trouble if people accidentally work on the full dataset after `.to_dask()`. But I am also fine with keeping 7 as a "sane" default for many analyses

Reply:
Thanks for the information; I was curious about that actually!